app/controllers/progress_job/progress_controller.rb

```diff
@@ -2,10 +2,14 @@ module ProgressJob
   class ProgressController < ActionController::Base
 
     def show
-      @delayed_job = Delayed::Job.find(params[:job_id])
-      percentage = !@delayed_job.progress_max.zero? ? @delayed_job.progress_current / @delayed_job.progress_max.to_f * 100 : 0
-      render json: @delayed_job.attributes.merge!(percentage: percentage).to_json
+      @delayed_job = Delayed::Job.find_by(id: params[:job_id])
```

Comment:
When I tried this branch, I was still getting some errors here, I think because of some issues with the file I was importing. Would it be possible to add some error checking to the `DelayedJob` lookup?